### DISCUSSION TITLE
wpf image控件大图(500M)显示特别慢,有什么优化的方法嘛？

### DISCUSSION THREAD

**Discussion by designerlast** [BYJRK/DotNet-Discussions#64] [WPF 开发]
还有path渲染的roi也占了一部分时间

**Comment by BYJRK** [comment]
ROI 是什么，Region Of Interest 吗？

**Reply by designerlast** [reply]
是的，在图片上显示的一般

**Reply by BYJRK** [reply]
你或许可以考虑用gdi直接绘制在图片上，而不是向界面添加额外的几何控件

**Comment by Dynesshely** [comment]
一般会先在后台计算一张小的缩略图, 符合 image 控件实际显示大小的图片
然后再交给 image 控件显示

**Reply by Dynesshely** [reply]
image 控件会原封不动的将你给的图片画到屏幕上, 500 MB 的大图可想而知
